### PR TITLE
Add explicit solvent host-guest branch

### PR DESCRIPTION
**Pull request by jchodera**
* Add explicit solvent
* Rename host to `CB7` to match `B2` guest
* Tweak some paths

Still testing.

### CODE REVIEW

**Comment by jchodera** [comment]
We should get this merged in!
Can someone review?

File: examples/binding/host-guest/run-torque.sh [hunk 1]
```diff
@@ -16,18 +16,20 @@
 #
 # nodes: number of nodes
 #   ppn: how many cores per node to use
-#PBS -l nodes=1:ppn=4:gpus=4:shared
+#PBS -l nodes=2:ppn=4:gpus=4:shared:gtx680
```
Comment (andrrizzi) [inline examples/binding/host-guest/run-torque.sh]: Should we remove forcing the gtx680?

File: examples/binding/host-guest/yank.yaml [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
   verbose: yes
-  number_of_iterations: 500
+  number_of_iterations: 100
   temperature: 300*kelvin
-  pressure: null # Run with NVT ensemble by setting to "null"
-  output_dir: hgoutput # Does not have to be set, but we do here to show the different outputs
-
+  pressure: 1*atmospheres
+  output_dir: output # Does not have to be set, but we do here to show the different outputs
+  platform: OpenCL
```
Comment (andrrizzi) [inline examples/binding/host-guest/yank.yaml]: Remove the platform?

File: examples/binding/host-guest/yank.yaml [hunk 3]
```diff
@@ -2,16 +2,16 @@
 options:
   minimize: yes
   verbose: yes
-  number_of_iterations: 500
+  number_of_iterations: 100
```
Comment (andrrizzi) [inline examples/binding/host-guest/yank.yaml]: Was lowering the number of iterations to only 100 intentional?

**Review by andrrizzi — APPROVED** [review]
Looks good beside the minor questions!

File: examples/binding/host-guest/yank.yaml [hunk 4]
```diff
@@ -29,32 +28,47 @@ solvents:
...
     solvent: GBSA
     leap:
       parameters: leaprc.gaff2
 
+  explicit:
+    receptor: CB7
+    ligand: B2
+    solvent: pme
+    leap:
+      parameters: [oldff/leaprc.ff14SB, leaprc.gaff, frcmod.ionsjc_tip3p]
```
Comment (Lnaden) [inline examples/binding/host-guest/yank.yaml]: Can you change the parameters to the following: 

`leaprc.protein.ff14SB, leaprc.gaff, leaprc.water.tip3p`

Comment (andrrizzi) [reply examples/binding/host-guest/yank.yaml]: Does `leaprc.water.tip3p` also contains the ions parameters that used to be in `frcmod.ionsjc_tip3p`?

Comment (Lnaden) [reply examples/binding/host-guest/yank.yaml]: `loadAmberParams frcmod.ionsjc_tip3p`

yes

**Comment by jchodera** [comment]
Addressing this now.

**Comment by jchodera** [comment]
Fixed!

**Review by Lnaden — APPROVED** [review]
lgtm

**Pull request merged**